pkg/templates/nstemplatetiers/nstemplatetier_generator.go

```diff
@@ -61,6 +57,8 @@ func CreateOrUpdateResources(s *runtime.Scheme, client client.Client, namespace
 		tmplTierKeys = append(tmplTierKeys, tmplTier)
 	}
 	sort.Strings(tmplTierKeys)
```

Comment:
could you please just remind me why we need the sorting? In other words, why we need to know in which order the NSTemplateTiers are created/updated...?

Reply:
sorting was for the tests, but since I'm using a regex to check the error message now, I can probably remove that part.

Reply:
removed in 3ae9ebc and using a regex to assert the error message